cicd/forgeops-tests/tests/metadata/test_idm_metadata.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2016-2019 ForgeRock AS. Use of this source code is subject to the
+# Common Development and Distribution License (CDDL) that can be found in the LICENSE file
+
+"""
+Verify the versions of ForgeRock Product in use.
+"""
+
+# Lib imports
+import os
+import pytest
+
+# Framework imports
+from utils import logger, kubectl, pod, idm_pod
+from utils.idm_pod import IDMPod
+
+
+class TestIDMMetadata(object):
```

Comment:
Do we not want to check:
* commons version
* JDK
* Legal notices

Reply:
Yes, to be implemented.  Was just looking for comments on initial code before implemented those.